js/bybit.js

```diff
@@ -4503,27 +4466,10 @@ module.exports = class bybit extends Exchange {
         //
         const result = this.safeValue (response, 'result', {});
         const chains = this.safeValue (result, 'chains', []);
-        const chainsLength = chains.length;
-        if (chainsLength === 0) {
-            const errorMessage = (networkId !== undefined) ? ('network ' + networkId + ' was not found for ' + code) : ('no deposit networks were found for ' + code);
-            throw new InvalidAddress (this.id + ' fetchDepositAddress() - ' + errorMessage);
-        } else {
-            const chainsIndexedById = this.indexBy (chains, 'chain');
-            let chosenNetworkId = undefined;
-            if (networkId !== undefined) {
-                if (networkId in chainsIndexedById) {
-                    chosenNetworkId = networkId;
-                } else {
-                    throw new InvalidAddress (this.id + ' fetchDepositAddress() - no deposit networks were found for ' + code);
-                }
-            } else {
-                const ids = Object.keys (chainsIndexedById);
-                const defaultNetwordId = this.defaultNetworkId (code);
-                chosenNetworkId = (defaultNetwordId in chainsIndexedById) ? defaultNetwordId : ids[0];
-            }
-            const addressInfo = chainsIndexedById[chosenNetworkId];
-            return this.parseDepositAddress (addressInfo, currency);
-        }
+        const chainsIndexedById = this.indexBy (chains, 'chain');
+        const selectedNetworkId = this.selectDefaultNetworkId (code, networkCode, chainsIndexedById);
+        const addressObject = this.safeValue (chainsIndexedById, selectedNetworkId, {});
```

Comment:
the above two lines replaced dozens of custom codes in CEX & bybit (other exchanges in the row of update).